layouts/partials/header.html

```diff
@@ -33,14 +33,14 @@
 			<div id="navbar" class="navbar-collapse collapse">
 				<ul class="nav navbar-nav">
 					<li {{ if eq $isHomePage true }}class="active"{{ end }}><a href="{{ $baseUrl }}/">Home</a></li>
-			{{ if isset .Site.Params "mainMenu" }}
+			{{ if .Site.Params.mainMenu }}
 				{{ $url := .Permalink }}
 				{{ range $menu := .Site.Params.mainMenu }}
 					{{ $itemUrl := printf "%s/%s/" $baseUrl $menu.link }}
 					<li {{ if eq $url $itemUrl }}class="active"{{ end }}><a href="{{ $baseUrl }}/{{ $menu.link }}">{{ $menu.name }}</a></li>
 				{{ end }}
 			{{ end }}
-				{{ if isset .Site.Params "noCategoryLink" }}{{ else }}
+				{{ if .Site.Params.noCategoryLink }}{{ else }}
```

Comment:
Consider instead `{{ if not .Site.Params.noCategoryLink }}` since nothing happens in the `if` block, anyway.